template/default

```diff
@@ -1,18 +1,29 @@
 {% if group.type == 'working group' or group.type == 'interest group'  %}
-Welcome to the W3C {{ group.name }}!
 
-The following link will lead you to a "facility map" 
-to essential information about this group and other W3C resources.
+ {% if group.type == 'working group' %}
+   {% set shorttype = 'wg' %}
+ {% else %}
+   {% set shorttype = 'ig' %}
+ {% endif %}
 
-Please take a few minutes to browse and bookmark this information:
-  https://w3c.github.io/Guide/participant/group.html?gid={{ group.id }}
+Welcome to the {{ group.name }}!
 
-If you have not already done so, link your GitHub account to your W3C account:
+This is a short introduction email to provide useful links about the group.
+You will find a lot of information about the {{ group.name }} linked from 
+  https://www.w3.org/groups/{{ shorttype }}/{{ group.shortname }}
+
+In particular, the following page tells you where the {{ group.name }} works, including repositories
+  https://www.w3.org/groups/{{ shorttype }}/{{ group.shortname }}/tools
+
+For meetings, check the group calendar at
+  https://auth.w3.org/?url=https://www.w3.org/groups/{{ shorttype }}/{{ group.shortname }}/calendar
```

Comment:
@plehegar, @caribouW3, I'm not so sure on the benefit to link to an auth.w3.org URI, as on the calendar it should already be clear that people need to log in if they want to see further information such as the joining details. e.g. it is currently written:

> **Joining Instructions**
> 
> Instructions are restricted to W3C users with Member access. You need to [log in](https://auth.w3.org/?url=https%3A%2F%2Fwww.w3.org%2Fevents%2Fmeetings%2Ffc01a32d-085d-486a-bae0-2411ef3a2225%2F20231018T120000%2F) to see them.

and

> **Log in to subscribe to this event**

Reply:
I made all those changes.